proteus/mprans/CLSVOF.h

```diff
@@ -878,8 +881,9 @@ namespace proteus
 		    // SAVE SOLUTION // for other models
 		    ///////////////////
 		    q_u[eN_k] = u;
-		    q_m[eN_k] = u; //porosity*u;
+		    q_m[eN_k] = u;
 		    q_H[eN_k] = Hnp1;
+		    q_mH[eN_k] = porosity*Hnp1; //porosity*H(\phi)=(1-q_vos)*H(\phi)
```

Comment:
Update CLSVOF to include porosity effects (also below)

Reply:
@manuel-quezada you checked the way you updated the porosity effects in CLSVOF, right?